inc/remindertranslation.class.php

```diff
@@ -0,0 +1,314 @@
+<?php
+/**
+ * ---------------------------------------------------------------------
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ * Copyright (C) 2015-2017 Teclib' and contributors.
+ *
+ * http://glpi-project.org
+ *
+ * based on GLPI - Gestionnaire Libre de Parc Informatique
+ * Copyright (C) 2003-2014 by the INDEPNET Development Team.
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * GLPI is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * GLPI is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with GLPI. If not, see <http://www.gnu.org/licenses/>.
+ * ---------------------------------------------------------------------
+ */
+
+if (!defined('GLPI_ROOT')) {
+   die("Sorry. You can't access this file directly");
+}
+
+
+/**
+ * ReminderTranslation Class
+ *
+ * @since version 9.4
+**/
+class ReminderTranslation extends CommonDBChild {
+
+   static public $itemtype = 'Reminder';
+   static public $items_id = 'reminders_id';
+   public $dohistory       = true;
+   static public $logs_for_parent = false;
+
+   static $rightname       = 'reminder_public';
+
+
+
+   static function getTypeName($nb = 0) {
+      return _n('Translation', 'Translations', $nb);
+   }
+
+
+   function getForbiddenStandardMassiveAction() {
+
+      $forbidden   = parent::getForbiddenStandardMassiveAction();
+      $forbidden[] = 'update';
+      return $forbidden;
+   }
+
+
+   /**
+    * @see CommonGLPI::getTabNameForItem()
+   **/
+   function getTabNameForItem(CommonGLPI $item, $withtemplate = 0) {
+
+      if (self::canBeTranslated($item)
+            && Session::getCurrentInterface() != "helpdesk") {
+         $nb = 0;
+         if ($_SESSION['glpishow_count_on_tabs']) {
+            $nb = self::getNumberOfTranslationsForItem($item);
+         }
+         return self::createTabEntry(self::getTypeName(Session::getPluralNumber()), $nb);
+      }
+
+      return '';
+   }
+
+
+   /**
+    * @param $item            CommonGLPI object
```

Comment:
Doc is not OK, see https://glpi-developer-documentation.readthedocs.io/en/master/codingstandards.html#parameters-documentation

Reply:
(on all methods)